Automattic/jetpack
# Jetpack Setup Wizard: Add Updates Question Page

#### Changes proposed in this Pull Request:

This PR builds on the work in #15429 by adding the updates question page.

Figma design desktop:
<img width="981" alt="image" src="https://user-images.githubusercontent.com/42627630/80524042-9cd9d080-8954-11ea-812e-5769550ca578.png">

Figma design mobile:
<img width="317" alt="image" src="https://user-images.githubusercontent.com/42627630/80524095-b4b15480-8954-11ea-8cf7-e62559856c26.png">

Current state desktop:
<img width="1117" alt="image" src="https://user-images.githubusercontent.com/42627630/80524131-c5fa6100-8954-11ea-8e84-1d2b5b71f0f7.png">

Current state mobile:
<img width="411" alt="image" src="https://user-images.githubusercontent.com/42627630/80524209-e0ccd580-8954-11ea-9134-470d571190fc.png">

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
Project: p1HpG7-8BF-p2
Design: pbtFPC-ol-p2

#### Testing instructions:





1. Add the following to the end of wp-config.php: add_filter( 'jetpack_show_setup_wizard', '__return_true' );
2. Visit the Jetpack plugin and visit the `/setup` page, then advance to the `/setup/updates` page.
3. Verify that the back/forward button works.
4. Verify that the page design matches the Figma.
5. Shrink the window and again verify that the page design matches the Figma.

#### Proposed changelog entry for your changes:
* None needed; the setup wizard project as a whole should have one entry.
